From a64e97dd476bda7c7981979d045b0d06d6f7ce47 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Mon, 2 Dec 2019 19:55:10 +0000
Subject: [PATCH] wallet: unbreak with boost 1.72

wallet/walletutil.cpp:77:23: error: no member named 'level' in 'boost::filesystem::recursive_directory_iterator'
        } else if (it.level() == 0 && it->symlink_status().type() == fs::regular_file && IsBerkeleyBtree(it...
                   ~~ ^
---
 src/fs.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/fs.h b/src/fs.h
index c713297d6ef0..8af81f173bdf 100644
--- src/fs.h
+++ src/fs.h
@@ -11,7 +11,6 @@
 #include <ext/stdio_filebuf.h>
 #endif
 
-#define BOOST_FILESYSTEM_NO_DEPRECATED
 #include <boost/filesystem.hpp>
 #include <boost/filesystem/fstream.hpp>
 
